placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -56,6 +58,29 @@
 
     private boolean completionEnabled = true;
 
+    private boolean clearEnabled;
+
+    private boolean justCleared;
```

Comment:
it doesn't seem like this value is ever read, can it be removed?

Reply:
Fixed removed